src/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, useRef } from 'react';
+// UNFORTUNATELY WHEN LIGHT VERSION IS USED, SOME ANIMATIONS ARE NOT WORKING AS EXPECTED
+// import lottie from 'lottie-web/build/player/lottie_light';
+import lottie from 'lottie-web';
+import type { AnimationItem, AnimationConfigWithData, AnimationConfig } from 'lottie-web';
+
+import { getEffect, getEvents } from './utils';
+import type { Animation, AnimationEffect } from './utils';
+
+type Props = {
+  animation: Animation;
+  reverse?: boolean;
+  strokeColor?: string;
+  pathCss?: string;
+  options?: Partial<AnimationConfig>;
+  size?: number;
+  loop?: AnimationConfig['loop'];
+  autoplay?: AnimationConfig['autoplay'];
+  wrapperStyle?: React.CSSProperties;
+} & React.HTMLProps<HTMLDivElement>;
+
+const UseAnimations: React.FC<Props> = ({
+  animation: { animationData, animationKey },
+  reverse = false,
+  size = 24,
+  strokeColor,
+  pathCss,
+  loop,
+  autoplay,
+  wrapperStyle,
+  options,
+  onClick,
+  ...other
+}) => {
+  const [animation, setAnimation] = useState<AnimationItem>();
+  const ref = useRef<HTMLDivElement>(null);
+  const getRandomId = (key: Animation['animationKey']) =>
```

Comment:
what do you think about moving this to `./util` folder?

Reply:
I'd usually go for util only if it would be used in more places. This is just a simple function for our specific case so I'd keep it there. If we would have more of this general functions, something like `util/helpers` is possible, but for one function, I think it's not necessary. But if you prefer to put it there, i have nothing against it.